### ISSUE TITLE
Hand tool doesn't scroll / pan when using stylus x201

### ISSUE THREAD

**Issue by Bugz000** [xournalpp/xournalpp#4064]
**Affects versions :**
 - OS: Debian 11 bullseye, KDE plasma 5
 - Version: xournal++ 1.1.1 
 - PC: lenovo thinkpad x201 TABLET

**Describe the bug**
the action/button is mapped, i press it, and move the stylus across the screen , does not scroll, pressing middle mouse button and moving mouse does work

**To Reproduce**
1. press stylus button
2. touch screen with stylus
3. move stylus across screen

**Expected behavior**
it scroll.

**Screenshots of Problem**
![2](https://user-images.githubusercontent.com/4687579/171061219-1b3c48aa-950b-4a3e-b4d9-a1c62c1ddc43.png)
![1](https://user-images.githubusercontent.com/4687579/171061222-1516cd90-81cb-4358-8c09-16d0a7458fae.png)

**Additional context**
#1501 #3948 #2293

**Comment by Bugz000** [comment]
yeah honestly i dont know where to begin with this software, having to set the colors PER PAGE, having to recompile the entire thing just to turn on logging, no dark mode, one pen shape, input issues that persist after being "fixed" how many times
can't use the floating menu without having toolbars on the side it's just a mess

mark this issue as solved, i've just installed openboard, miles better, (dark mode available as stock too, it's absolutely essential in this day and age imo)

**Comment by codewhizz** [comment]
chill dude.

**Comment by Bugz000** [comment]
> chill dude.

i don't know where you sensed aggression, but i apologise if i caused alarm

however i must amend this bug report - in another piece of software the pen buttons were not being picked up, i installed another package recommended by their support channels, it was something wacom, and sure enough it started detecting the pens, i guess there's a dependency missing, would be good to add it, i can dig it up if the devs care enough
the rest however still stands, software does need some serious work

i've also since moved from openboards so for those reading and if the devs of xournal++ wish for some non partial inspiration, i now use stylus labs write, it fits all the bills, though i wish the "wide" pen was rotated 45 instead of straight vertical, it's absolutely close enough, everything else is perfect :)

**Comment by LittleHuba** [comment]
Thank you very much for your input. If you should ever have the time to also follow up on your constructive criticism, feel free to make logging available via checkbox or add that simple fix for the tool. We are always open for contributions.

Until then, I hope you are happy with other solutions out there.

**Comment by rolandlo** [comment]
If anyone else experiences the hand scrolling issue (and enabling the `Merge button events with stylus tips events` checkbox like in https://github.com/xournalpp/xournalpp/issues/3948#issuecomment-1368248546 doesn't help), please provide [input logs](https://github.com/xournalpp/xournalpp/wiki/How-to-provide-debug-info-for-input-related-bugs)
